psci.el

```diff
@@ -232,14 +190,13 @@ We chose to load the .psci file's content (the purescript doc proposes its use).
 (defvar inferior-psci-mode-map
   (let ((map (make-sparse-keymap)))
     (define-key map (kbd "C-c C-l") 'psci/load-current-file!)
-    (define-key map (kbd "C-c C-r") 'psci/load-project-modules!)
     (define-key map (kbd "C-c M-n") 'psci/load-module!)
     map)
   "Basic mode map for `inferior-psci-mode'.")
 
 (defgroup psci nil "psci customisation group."
   :tag "psci"
-  :version "0.0.4"
+  :version "0.0.9"
```

Comment:
Ok, didn't realize that.  Thought it just corresponded to the psci version, which I assumed to be 0.0.9 for the 0.9.x compiler.  Will change back